superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -323,19 +323,19 @@ export default class FilterableTable extends PureComponent<
 
   sortResults(sortBy: string, descending: boolean) {
     return (a: Datum, b: Datum) => {
-      const aValue = a[sortBy];
-      const bValue = b[sortBy];
-      if (aValue === bValue) {
-        // equal items sort equally
-        return 0;
-      }
-      if (aValue === null) {
-        // nulls sort after anything else
-        return 1;
-      }
-      if (bValue === null) {
-        return -1;
-      }
```

Comment:
This is still needed, right? So that `null` sort after non-null values?

Reply:
I got excited and forgot to add the NULL handling back 😅 
Added back in [`this commit`](https://github.com/apache/superset/pull/17573/commits/580f9d9671d831a0c00fd1b0ee54aadbee90faf5).